apps/api/src/server/applications/application/application.routes.js

```diff
@@ -39,6 +40,29 @@ router.post(
 	asyncHandler(createApplication)
 );
 
+router.get(
+	'/',
+	/*
+        #swagger.tags = ['Applications']
+        #swagger.path = '/applications'
+        #swagger.description = 'Returns the application by its string reference'
+        #swagger.parameters['reference'] = {
+            in: 'query',
+            description: 'Application reference',
+            required: false,
+            type: 'string'
+        }
+        #swagger.responses[200] = {
+            description: 'Application Details',
+            schema: { id: 1, reference: 'AB0110203', status: 'Pre-Application'}
```

Comment:
We could do with a proper swagger schema for Application - but that's beyond the scope of this.

e.g. subscription example: https://github.com/Planning-Inspectorate/back-office/blob/main/apps/api/src/server/swagger.js#L1463